src/main/java/org/breedinginsight/daos/ProgramDAO.java

```diff
@@ -114,7 +131,13 @@ public ProgramDAO(Configuration config, DSLContext dsl, BrAPIProvider brAPIProvi
 
         // Parse the result
         for (Record record: queryResult){
+            if (record.getValue(PROGRAM.BRAPI_URL) == null) {
+                record.setValue(PROGRAM.BRAPI_URL, SYSTEM_DEFAULT);
+            }
```

Comment:
Would it be better to store the default brapi url with the program? The scenario I'm thinking of is if we have default system A and create a Program 1, but then we switch our default system to system B, Program 1 will be looking in default system B for its data, but it will not be there. 

Edit: I see this was addressed below, but leaving my comment above in case it is talking about a different thing. What was the decision for this? To fix it now, or leave it for later?

Reply:
Ya, talked about this with Tim and decided to leave it as-is for now.